### ISSUE TITLE
what is the name of MVC framework in vendor/javascript dir?

### ISSUE THREAD

**Issue by hwh008** [maccman/monocle#24]
I am new to ruby and coffee, I try to learn from monocle source.
now I am confuse with the coffee MVC framework, so what is it?

**Comment by raybesiga** [comment]
@hwh008 The MVC framework that Monocle is using is [Sinatra](http://www.sinatrarb.com/intro.html) and the [Coffeescript](http://coffeescript.org/) is simply a language that compiles to Javascript. It is using a gem that ends in the .coffee extension but it all maintains the basic MVC architecture.

**Comment by hwh008** [comment]
@raybesiga Thanks, man.
I see the Monocle is using Sinatra and Coffee, but my question is about the client side's MVC framework, something like `model.module.coffee`, `controller.module.coffee` and `router.module.coffee`.
I read the code, maybe it is like what you said, the basic MVC without name.

**Comment by maccman** [comment]
Yeah, it's just a little toy MVC framework I've built and use for all my
projects :)

On Tue, Feb 18, 2014 at 1:36 AM, hwh008 notifications@github.com wrote:

> @raybesiga https://github.com/raybesiga Thanks, man.
> I see the Monocle is using Sinatra and Coffee, but my question is about
> the client side's MVC framework, something like model.module.coffee,
> controller.module.coffee and router.module.coffee.
> I read the code, maybe it is like what you said, the basic MVC without
> name.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/maccman/monocle/issues/24#issuecomment-35341678
> .

## 

Alex MacCaw

+1.415.624.7665
@maccaw

http://alexmaccaw.com

**Comment by raybesiga** [comment]
@maccman It is so neat! You should do a little write up for those of us who wish to take it up.
